Hudl.FFmpeg.Core/Extensions/ProcessExtensions.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Diagnostics;
+using System.Threading;
+
+namespace Hudl.FFmpeg.Extensions
+{
+    public static class ProcessExtensions
+    {
+        public static bool WaitForProcessStart(this Process process)
+        {
+            return process.WaitForProcessStart(null);
+        }
+        public static bool WaitForProcessStart(this Process process, int? timeoutMilliseconds)
```

Comment:
Why go with an optional timeout for this method? I personally would make timeout required and then remove the default 10 second timeout.